lib/templates/pullRequestOpened.hbs

```diff
@@ -1,36 +1,36 @@
 
-If this is your first contribution, please send in a [Contributor License Agreement](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#contributor-license-agreement-cla) (CLA) so that we can review and merge this pull request.
+The [Pull Request Guidelines](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines) is a handy reference for making sure your PR gets accepted quickly, so make sure to skim that.
 
-Maintainers, there was an error processing the CLA, resolve it to continue CLA checking.
-```
-{{ errorCla }}
-```
-{{else}}
-{{#if askForCla}}
-Welcome to the Cesium community @{{ userName }}!
+Let's see if this is ready for review.
 
-Can you please send in a [Contributor License Agreement](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#contributor-license-agreement-cla) (CLA) so that we can review and merge this pull request?
+* ❌ Missing `CONTRIBUTORS.md` entry.
+  * Please add yourself to the [contributors]({{ contributorsUrl }}) file! 
 {{else}}
-@{{ userName }}, thanks for the pull request! Maintainers, we have a signed CLA from @{{ userName }}, so you can review this at any time.
-{{/if}}
+Thanks for the pull request @{{ userName }}! Let's see if this is ready for review.
 {{/if}}
+{{#if claEnabled}}
+{{#if errorCla}}
+* ⚠ There was an error checking the CLA! If this is your first contribution, please send in a [Contributor License Agreement](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#contributor-license-agreement-cla).
+  * Maintainers, this was the error I ran into while attempting to process the CLA check. Please resolve it to continue CLA checking.
+  ```
+  {{ errorCla }}
+  ```
 {{else}}
-@{{ userName }}, thanks for the pull request!
+{{#if askForCla}}
+* ❌ Missing CLA. 
+  * Please send in a [Contributor License Agreement](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#contributor-license-agreement-cla) (CLA) and comment back here to let us know to check this!
+{{else}}
+* ✔ Signed CLA found. 
+{{/if}}
 {{/if}}
-
-{{#if askAboutContributors}}
-I noticed you haven't added yourself to `CONTRIBUTORS.md`. Please add yourself to that file and comment back here to let us know it's ready! 
-
 {{/if}}
-
 {{#if askAboutChanges}}
-I noticed that [CHANGES.md]({{ repository_url }}/blob/master/CHANGES.md) has not been updated. If this change updates the public API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and comment on this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).
-
+* ❔ [CHANGES.md]({{ repository_url }}/blob/master/CHANGES.md) was not updated.
```

Comment:
Instead of master, can we link to `CHANGES.md` for this branch? That will let the contributor click on that link and edit it in the browser.